### PR TITLE
-fix non-diagonal noise for LambaEM and RKMiL_General

### PR DESCRIPTION
**Pull request by KirillZubov**
-fix non-diagonal noise for LambaEM and RKMiL_General
-support RKMiL_General with Float32

### CODE REVIEW

File: src/iterated_integrals.jl [hunk 1]
```diff
@@ -186,10 +186,11 @@ function get_iterated_I!(dt, dW, Wik::WikJGeneral_oop, C=1)
...
 
     WikJ = dW*dW'
-    Gp₁ = randn(M)
-    α = sqrt(1 + sum_dW²/dt)
-    Gp₂ = Gp₁/(sqrt(2)*(1+α)*dt)
-
+    eltWikJ = eltype(WikJ)
+    WikJ2 =zeros(eltWikJ, size(WikJ))
+    Gp₁ = randn(eltype(WikJ),M)
+    α = eltype(WikJ)(sqrt(1 + sum_dW²/dt))
```
Comment (ChrisRackauckas) [inline src/iterated_integrals.jl]: use convert instead of implicit number constructor

File: src/perform_step/low_order.jl [hunk 2]
```diff
@@ -249,6 +249,8 @@ end
   du₁ = integrator.f(uprev,p,t)
   L = integrator.g(uprev,p,t)
   mil_correction = zero(u)
+  ggprime_norm = zero(eltype(u)) #0
+  # sqdt = integrator.sqdt
```
Comment (ChrisRackauckas) [inline src/perform_step/low_order.jl]: what's the reason here?

File: src/iterated_integrals.jl (hunk 1 above)
Comment (KirillZubov) [reply src/iterated_integrals.jl]: ok

File: src/perform_step/low_order.jl (hunk 2 above)
Comment (ChrisRackauckas) [reply src/perform_step/low_order.jl]: no, the `sqdt=integrator.sqdt` getting commented, and then using the integrator in the other lines? Seems like an odd change.

**Pull request merged**